src/tespy/components/heat_exchangers.py

```diff
@@ -867,6 +867,26 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a simple heat exchanger.
+
+            .. math::
+
+                T_m = \frac{T_{ein} - T_{aus}}{\ln\left({\frac{T_{ein}}
+                                                         {T_{aus}}}\right)}
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \left(1 - \frac{T_{amb}}
+                                   {T_m}\right) \cdot \dot{Q}
+                Ex_{loss} = 0
+        """
+        Tm = (self.inl[0].T.val_SI - self.outl[0].T.val_SI) / (
+            np.log(self.inl[0].T.val_SI / self.outl[0].T.val_SI))
+        self.Ex_output = self.inl[0].m.val_SI * (self.outl[0].ex_physical
+                                                 - self.inl[0].ex_physical)
+        self.Ex_input = (1 - (Tamb / Tm)) * self.Q.val
+        self.Ex_loss = 0
```

Comment:
The exergy loss is not necessarily 0, I think. What about e.g. pressure losses when no heat is transferred? My suggestions would be:

- Q > 0 -> `self.Ex_input = (1 - (Tamb / Tm)) * self.Q.val`, `self.Ex_output = self.inl[0].m.val_SI * (self.outl[0].ex_physical - self.inl[0].ex_physical)` and `self.Ex_loss = self.Ex_input - self.Ex_output`
- Q = 0 -> `self.Ex_input = 0`, `self.Ex_output = 0` and `self.Ex_loss = - self.inl[0].m.val_SI * (self.outl[0].ex_physical - self.inl[0].ex_physical)`
- Q < 0 -> `self.Ex_input = self.inl[0].m.val_SI * (self.outl[0].ex_physical - self.inl[0].ex_physical)`, `self.Ex_output = (1 - (Tamb / Tm)) * self.Q.val` and `self.Ex_loss = self.Ex_output - self.Ex_input`

What do you think?